deploy/helm-chart/README.md

```diff
@@ -0,0 +1,135 @@
+```
+
+## Upgrading the Chart
+### Upgrading the Gateway resources
+Helm does not upgrade CRDs during a release upgrade, or on an install if they are already present in the cluster.
+Before you upgrade a release, ensure the Gateway API resources are up to date by doing one of the following:
+
+1. To upgrade the Gateway resources from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
+   ```
+   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.7.1/standard-install.yaml
+   ```
+1. To upgrade the Gateway resources from the NGINX Kubernetes Gateway Chart sources, pull the chart sources as described
+   in [Pulling the Chart](#pulling-the-chart) and then run:
+   ```
+   kubectl apply -f crds/
+   ```
+
+>Note: The following warning is expected and can be ignored: `Warning: kubectl apply should be used on resource created
+by either kubectl create --save-config or kubectl apply`.
+
+### Upgrading the Chart from the OCI registry
+To upgrade the release `my-release`, run:
+
+```
+helm upgrade my-release oci://ghcr.io/nginxinc/charts/nginx-ingress --version 0.0.0-edge -n nginx-gateway
+```
+
+### Upgrading the Chart from the sources
+
+Pull the chart sources as described in [Pulling the Chart](#pulling-the-chart), if not already present. Then, to upgrade
+the release `my-release`, run:
+```
+helm upgrade my-release . -n nginx-gateway
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the release `my-release`:
+
+```
+helm uninstall my-release -n nginx-gateway
+```
+
+The command removes all the Kubernetes components associated with the release and deletes the release.
+
+### Uninstalling the Gateway resources
+Uninstalling the release does NOT uninstall the CRDs or validating webhook. To clean up these resources, do one of the
+following:
+
+>**Warning: These commands will delete all the corresponding custom resources in your cluster across all namespaces!
+Please ensure there are no custom resources that you want to keep and there are no other Gateway API implementations
+running in the cluster!**
+
+1. To delete the Gateway resources using [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
+   ```
+   kubectl delete -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.7.1/standard-install.yaml
+   ```
+1. To delete the Gateway resources using the NGINX Kubernetes Gateway Chart sources, pull the chart sources as described
+   in [Pulling the Chart](#pulling-the-chart) and then run:
+   ```
+   kubectl delete -f crds/
+   ```
+
+## Configuration
+
+The following tables lists the configurable parameters of the NGINX Kubernetes Gateway chart and their default values.
+
+|Parameter | Description | Default |
+| --- | --- | --- |
+|`nginxGateway.image.repository` | The repository for the NGINX Kubernetes Gateway image. | ghcr.io/nginxinc/nginx-kubernetes-gateway |
```

Comment:
nit: can we use `fieldName` and `fieldValue` (including the default column) formatting in the table improve readability?

Reply:
I'm not exactly sure what you mean here 🤔

Reply:
For example:

|`nginxGateway.image.pullPolicy` | The `imagePullPolicy` for the NGINX Kubernetes Gateway image. | `Always` |

Reply:
Got it, thanks for the clarification! Done for the fields in the pod spec. I'm not sure if I find the same formatting easier to read for the value in the default column so  I haven't changed that just yet - should it be all values? I personally prefer it without the back ticks, but happy to be overruled 😁